pom.xml

```diff
@@ -12,56 +12,77 @@
         <description>IUCDS pilot AWS Middleware</description>
 
         <parent>
-                <groupId>com.cooldatasoft</groupId>
+                <groupId>org.springframework.boot</groupId>
                 <artifactId>spring-boot-starter-parent</artifactId>
-                <version>2.2.5.RELEASE</version>
+                <version>2.3.0.RELEASE</version>
         </parent>
 
         <properties>
                 <docker.image.prefix>nhsd</docker.image.prefix>
                 <java.version>11</java.version>
                 <aws-sdk.version>1.11.362</aws-sdk.version>
                 <jacoco.line.coveredRatio>0.65</jacoco.line.coveredRatio>
+                <spotbugs.version>4.0.0</spotbugs.version>
+                <maven.project.info.reports.plugin.version>2.9</maven.project.info.reports.plugin.version>
         </properties>
 
         <build>
-                <plugins>
-                        <plugin>
-                                <groupId>org.apache.maven.plugins</groupId>
-                                <artifactId>maven-compiler-plugin</artifactId>
-                                <configuration>
-                                        <source>11</source>
-                                        <target>11</target>
-                                </configuration>
-                        </plugin>
-                        <plugin>
-                                <groupId>org.springframework.boot</groupId>
-                                <artifactId>spring-boot-maven-plugin</artifactId>
-                        </plugin>
-                        <plugin>
-                                <groupId>com.spotify</groupId>
-                                <artifactId>dockerfile-maven-plugin</artifactId>
-                        </plugin>
-                        <plugin>
-                                <groupId>org.apache.maven.plugins</groupId>
-                                <artifactId>maven-dependency-plugin</artifactId>
-                        </plugin>
-                        <plugin>
-                                <groupId>org.apache.maven.plugins</groupId>
-                                <artifactId>maven-surefire-report-plugin</artifactId>
-                                <configuration>
-                                        <argLine>-Xmx1024M ${argLine}</argLine>
-                                </configuration>
-                        </plugin>
-                        <plugin>
-                                <groupId>org.jacoco</groupId>
-                                <artifactId>jacoco-maven-plugin</artifactId>
-                        </plugin>
-                        <plugin>
-                                <groupId>org.codehaus.mojo</groupId>
-                                <artifactId>versions-maven-plugin</artifactId>
-                        </plugin>
-                </plugins>
+                <pluginManagement>
```

Comment:
plugins needs to stay inside the plugins tag rather than the pluginManagement tag otherwise they wont run as part of the build.